### DISCUSSION TITLE
Where is the discord server?

### DISCUSSION THREAD

**Discussion by ssbarnea** [go-task/task#742] [Q&A] [answered]
Two weeks ago a link to discord server was added to the docs and today clicking it gives:

> Invite Invalid
> This invite may be expired, or you might not have permission to join.

Also searching on discord for public servers with "taskfile" or "task" does not return any useful results.

Was the IM server plan already canceled?

**Comment by andreynering** [comment, marked as answer]
Hi @ssbarnea,

I didn't know those links can expire 🤦

I just created a new (now unlimited) one: https://discord.gg/6TY36E39UK

Updated the website as well: d35141a369b5bbe3b06d02261f3499ccfbc9c276
